test/unit/api/ws/workers/peersUpdateRules.js

```diff
@@ -45,7 +40,7 @@ describe('PeersUpdateRules', function () {
 		validConnectionId  = 'ABCDEF123456789';
 		validErrorCode = 4100;
 		peersUpdateRules = new PeersUpdateRules(slaveWAMPServerMock);
-		peersUpdateRules.slaveToMasterSender.send = sinon.stub(peersUpdateRules.slaveToMasterSender, 'send').callsArg(3, null);
+		peersUpdateRules.slaveToMasterSender.send = sinonSandbox.stub(peersUpdateRules.slaveToMasterSender, 'send').callsArg(3, null);
```

Comment:
This assignment is an odd pattern. I don't think it does anything not covered by `sinonSandbox.stub(peersUpdateRules.slaveToMasterSender, 'send').callsArg(3, null);` on its own